eclair-core/src/main/scala/fr/acinq/eclair/wire/protocol/LightningMessageCodecs.scala

```diff
@@ -417,7 +417,7 @@ object LightningMessageCodecs {
 
   val unknownMessageCodec: Codec[UnknownMessage] = (
     ("tag" | uint16) ::
-      ("message" | varsizebinarydata)
+      ("message" | bytes)
```

Comment:
Yep, this is what this codec should have done from the start, there is no good reason to require unknown messages to start with their length (especially a 2-bytes length which is incompatible with a TLV stream). You're not breaking compatibility here, plugins will simply have to separately read the first two bytes if they expect a length prefix :+1: